composer.json

```diff
@@ -57,7 +57,7 @@
         "zumba/amplitude-php": "^1.0.4"
     },
     "require-dev": {
-        "acquia/coding-standards": "^3.0.1",
+        "acquia/coding-standards": "^3.0.2",
```

Comment:
This isn't directly related to the PR; it was necessary to fix breaking changes in a recent release of PHPUnit